ReactNativeClient/lib/components/screens/encryption-config.js

```diff
@@ -131,6 +138,9 @@ class EncryptionConfigScreenComponent extends BaseScreenComponent {
 			try {
 				const password = this.state.passwordPromptAnswer;
 				if (!password) throw new Error(_('Password cannot be empty'));
+				const password2 = this.state.passwordPromptConfirmAnswer;
+				if (!password2) throw new Error(_('Confirm password cannot be empty'));
+				if (password !== password2) throw new Error(_('Passwords did not match!'));
```

Comment:
Please change `did` to `do`.

Reply:
Hmm, it shows 3 lines, instead of the one I clicked. So my previous comment is not very clear.
Please change: `Passwords did not match!` to `Passwords do not match!`

Reply:
Also please change it in the CLI app (if you rebase, you'll get your latest changes).

Reply:
@laurent22 no longer possible, since it is already merged. I just made the change for CLI myself 74c2bbc2

Reply:
@laurent22 ah, you meant in this PR. yep, that would have worked ;-) haha, my bad. I fixed it already for the CLI app.